docs/source/geo_def.rst

```diff
@@ -1,41 +1,41 @@
 Geometry definitions
 ====================
-The module **pyresample.geometry** contains classes for describing different kinds of types
-of remote sensing data geometries. The use of the different classes is described below.
+The module :mod:`pyresample.geometry <geometry>` contains classes for describing different kinds
```

Comment:
Does this make the text say "pyresample.geometry" and link to a document called "geometry" or does it make the text say "geometry" linked to "pyresample.geometry"?

If the latter I think this can be changed to ":mod:`~pyresample.geometry`".

Reply:
The functionality is :any:`Name_Displayed <Path_To_Thing>`. So it displays "pyresample.geometry" and links to the geometry module.